zbus/src/address/transport/unixexec.rs

```diff
@@ -48,7 +59,7 @@ impl<'a> TryFrom<&'a Address<'a>> for Unixexec<'a> {
 
     fn try_from(s: &'a Address<'a>) -> Result<Self> {
         let mut path = None;
-        let mut argv = Vec::new();
+        let mut args = HashMap::new();
```

Comment:
This brings in extras allocation. Why didn't you make use of the API I recommended that gives you de-duplication/resorting for free in the ideal case, even?

Reply:
If you mean using the `Vec::dedup_by` method, that would keep the *first* duplicated value and you mentioned you would like to only keep the *last* one.

Reply:
True. I'm sorry for the miscommunication. My main point was that we just pick one of the values if there are duplicates. The last one makes most sense but it's not necessary.

Reply:
Makes sense, I just wasn't sure if the "last value" was a hard requirement. I've pushed another revision that chooses the first value instead which got rid of the extra `HashMap` allocation.

Reply:
Nice! again, I'm sorry for not communicating well.